app/controllers/TagManagementApi.scala

```diff
@@ -37,50 +39,54 @@ object TagManagementApi extends Controller with PanDomainAuthActions {
     }
   }
 
-  def createTag() = (APIAuthAction andThen CreateTagPermissionsCheck).async { req =>
-    implicit val username = Option(req.user.email)
-    req.body.asJson.map { json =>
-      json.as[CreateTagCommand].process.map { result =>
-        result.map{t => Ok(Json.toJson(DenormalisedTag(t))) } getOrElse NotFound
-      } recover {
-        commandErrorAsResult
+  def createTag = CORSable(conf.corsablePostDomains: _*) {
+    (APIAuthAction andThen CreateTagPermissionsCheck).async { req =>
+      implicit val username = Option(req.user.email)
+      req.body.asJson.map { json =>
+        json.as[CreateTagCommand].process.map { result =>
+          result.map { t => Ok(Json.toJson(DenormalisedTag(t))) } getOrElse NotFound
+        } recover {
+          commandErrorAsResult
+        }
+      }.getOrElse {
+        Future.successful(BadRequest("Expecting Json data"))
       }
-    }.getOrElse {
-      Future.successful(BadRequest("Expecting Json data"))
     }
   }
 
-  def searchTags = APIHMACAuthAction { req =>
+  def searchTags = CORSable(conf.corsablePostDomains: _*) {
+    APIHMACAuthAction { req =>
```

Comment:
Sorry for all the horrible spacing diffs here - the only change is this line: `def searchTags = CORSable(conf.corsablePostDomains: _*) {`